go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/ethereum/go-ethereum v1.14.12
 	github.com/gogo/protobuf v1.3.2
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
-	github.com/hyperledger-labs/yui-relayer v0.5.11
+	github.com/hyperledger-labs/yui-relayer v0.5.12
```

Comment:
As you can see in the [first commit](https://github.com/datachainlab/lcp-go/pull/43/commits/398b42cdd60e863c8ee82eb3664d57d74ad803d9), the signatures of some methods in the tendermint module have changed.